.github/ISSUE_TEMPLATE/RFC_report.md

```diff
@@ -0,0 +1,87 @@
+---
+name: RFC template
+about: request for comments
+title: "[RFC]"
+labels: 'RFC'
+assignees: ''
+
+---
```

Comment:
The template looks fine to me.

A quick search on Google shows that this template origins from [freechipsproject/rfcs](https://github.com/freechipsproject/rfcs/blob/master/000-template.md). As a courtesy to the original authors and also an obligation to the copyright license (which is Apache License), we need to add the License file somewhere.

Reply:
This template is well-used. For example https://github.com/rust-lang/lang-team/blob/master/projects/declarative-macro-repetition-counts/0000-draft-rfc.md
I do not see such license used on those copies. So I think do not need is either.

Reply:
After doing some more googling, I found the RFC template is indeed used in several other projects. And we cannot know the original author of the text. So I guess it is ok to assume the text is in public domain and free to use.